__tests__/pages/index.test.tsx

```diff
@@ -3,7 +3,7 @@
  */
```

Comment:
I had to move this file because NextJS wants _everything_ inside `/pages` to actually be a page